mantidproject/mantid
# Vesuvio - VesuvioTOFFit incorrect params output

VesuvioTOFFit now returns the parameters from the initial Fit rather than those of the secondary fit that is used to calculate Chi Squared.

**To test:**

**I'd recommend building this in Release mode. The Vesuvio Algorithm, can run very slowly in Debug**
- Run the `Vesuvio_intensity_constraints_test.py` script available from `\\olympic\babylon5\Public\ElliotOram\Vesuvio` in the mantidplot scripting window with the DataArchive searching enabled.
  - You will also need the `IP_2015_06_Chimney_header.txt` in a directory where mantid can find it (this is in the same babylon5 directory)
- Inspect the data in the `23115-23153_params_iteration_1`
  - Ensure that `f4.Intensity / f3.Intensity = ~4.3437...` for each spectrum (`3`,`47`,`91`)
- Plot the data and fit from `23115-23153_data_backwards_bank_x_iteration_x` and check the fit looks reasonable.

Fixes #16039 

_Does not need to be in the release notes. As this has only been discovered in the nightly builds_

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
